jobs/config.json

```diff
@@ -8068,6 +8068,7 @@
       "--extract=ci/latest",
       "--gcp-zone=us-central1-f",
       "--kubeadm=ci",
+      "--kubelet=ci/latest",
```

Comment:
Ok for now, but this should probably be sth like `kubernetes-anywhere-kubelet-version`

Reply:
This would be preferable if it is possible to dynamically change that `kubernetes-anywhere-kubelet-version` arg dynamically in the kubernetes_e2e scenario. WDYT? Not blocking on that now though...

Reply:
If kubernetes-anywhere-kubelet-version is specified, it will use whatever is provided (ie. a gs bucket). If it is provided indirectly (via kubelet param) then it goes through the code to resolve the version provided to something k8s anywhere will understand (ie. a gs bucket).

What test scenarios are you trying to accomplish with a dynamic kubernetes-anywhere-kubelet-version test arg?